src/services/ui/serverless.yml

```diff
@@ -9,6 +9,11 @@ plugins:
   - serverless-stack-termination-protection
   - "@stratiformdigital/serverless-s3-security-helper"
 
+constructs:
+  react:
+    type: single-page-app
+    path: dist
+
```

Comment:
do we need this piece, im not sure what its doing

Reply:
Oh. That needs to go. Good call